test/integration/user/api/interactivity.test.js

```diff
@@ -70,7 +70,7 @@ describe('Interactivity', () => {
                     expect(event.features[0].style.color.blendTo).toBeDefined();
                     expect(event.features[0].style.color.reset).toBeDefined();
                     expect(event.features[0].style.reset).toBeDefined();
-                    expect(event.features[0]).toEqual({ id: 0, layerId: 'layer', properties: { cartodb_id: 0 }, style: jasmine.any(Object) });
+                    expect(event.features[0]).toEqual({ id: 0, layerId: 'layer', properties: { }, style: jasmine.any(Object) });
```

Comment:
I wouldn't filter it

Reply:
I think `cartodb_id`should not appear in properties because it is an internal way to achieve the feature.id. Since feature.id === feature.properties.cartodb_id we are duplicating always the information

Reply:
I disagree, but we can always change it later if we really found it's better, I won't worry now :+1: